Sources/PackageRegistry/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(PackageRegistry
+  RegistryConfiguration.swift)
+target_link_libraries(PackageRegistry PUBLIC
+  TSCBasic
+  PackageLoading
+  PackageModel
+  TSCUtility)
```

Comment:
Do we need `Basics` and `SourceControl` too?

Reply:
For just this command, all we need is Foundation. For the `RegistryManager` implementation in #3640, we'll also need `Basics`. I just made that change in 347926a1.